doc/examples/features_detection/plot_gabors_from_astronaut.py

```diff
@@ -51,7 +51,7 @@
 patch_shape = 8, 8
 n_filters = 49
 
-astro = color.rgb2gray(data.astronaut())
+astro = np.ascontiguousarray(color.rgb2gray(data.astronaut()))
```

Comment:
@emmanuelle Yes, the implementation there is not quite the most efficient and extracts a final slice on the color-transformed image.